### DISCUSSION TITLE
What's the reason for the speed limiter?

### DISCUSSION THREAD

**Discussion by khanv1ct** [crunchy-labs/crunchy-cli#347] [General]
Will CR flag or ban you for too many downloads in a period of time? The unrestricted speeds are crazy (1.32GB in a few seconds).

**Comment by bytedream** [comment]
This was a user request, see #250. I haven't heard of a ban yet, but sometimes there is a rate limit which kicks in (but idk what the conditions are for it to be triggered, it's months ago I had it the last time)

**Reply by khanv1ct** [reply]
Thanks for the reply. Awesome tool btw!
